### DISCUSSION TITLE
Same simulation result for different current settings?

### DISCUSSION THREAD

**Discussion by HanyangRuan** [simnibs/simnibs#247] [9. Results and analysis]
Dear SimNIBS experts,
When simulating the tDCS electric field, I mistakenly set the current direction for the cathode and anode incorrectly. However, when I adjusted to the correct current settings, I found that there was not a significant difference between the simulation results. 
![Screenshot from 2023-11-27 16-04-05](https://github.com/simnibs/simnibs/assets/152194240/3968bdfa-e819-41e9-b0ff-08ee24cf1902)
![Screenshot from 2023-11-27 16-01-10](https://github.com/simnibs/simnibs/assets/152194240/1ca1da6b-79fe-44ea-9b3d-18dd0c59b085)
My anode size is 4×4cm, cathode is 5×5cm, electrode thickness is 2mm, and gel thickness is 2mm. The current of anode is 2mA, and -2mA for cathode. Is there any issue in the setup process?
Thanks, Hanyang

**Comment by TorgeW** [comment]
For the electric field magnitude the polarity of the electrodes does not matter. The e field direction changes.  :)
